.idea/rubyWebdriver.iml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should add the `.idea` folder to a global `.gitignore` file on your machine, so that it doesn't get included everywhere. http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

Reply:
ohhhh i was wondering about this. should i make a separate PR for this?

Reply:
It doesn't actually require a PR. The global .gitignore doesn't get checked in anywhere. But once you create it, you can then update this PR to not include your .idea folder.

Reply:
Some additional future reading you can do:

https://driesvints.com/blog/getting-started-with-dotfiles/

I have all of my dotfiles (.gitconfig, .gitignore, .zshrc, .bashrc) synced to my dropbox, and have an easy way to set them up when I get a new computer.

Reply:
updated to a global .gitignore_global file!